.github/workflows/publish-to-dockerhub.yml

```diff
@@ -29,15 +28,17 @@ jobs:
       - name: Get tags
         run: git fetch --tags origin
 
-      - name: Login to DockerHub
+      - name: Login to GitHub Container Registry
         uses: docker/login-action@v3
         with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_TOKEN }}
+          registry: ghcr.io
+          username: ${{ secrets.GH_USERNAME }}
+          password: ${{ secrets.GH_PAT}}
+
       - name: Push image
         env:
-          DOCKERHUB_ACCOUNT: ${{ secrets.DOCKERHUB_ACCOUNT }}
-          DOCKERHUB_REPO: ${{ secrets.DOCKERHUB_REPO }}
+          GITHUB_ACCOUNT: ${{ secrets.GITHUB_ACCOUNT }}
+          GH_CONTAINER_REGISTRY_REPO: ${{ secrets.GH_CONTAINER_REGISTRY_REPO }}
```

Comment:
Could this also default to something like the name of this current repo?

Reply:
Yeah, sure!